### ISSUE TITLE
Scaling up ReplicaSet with one member results in member RECOVERING state

### ISSUE THREAD

**Issue by andrii-litvinov** [mongodb/mongodb-enterprise-kubernetes#73]
I have another weird issue that I don't know how to pinpoint. I have 1-member ReplicaSet that I want to scale up. When I scale it up to 2 or 3 members first member gets into RECOVERING state, cluster becomes unhealthy and cannot recover on its own.

To resolve I have to
- turn MongoDB into a Standalone deployment
- create role in CloudManager to allow removing records from `local.system.replset`
- remove replica set from `local.system.replset` collection
- change cluster back to ReplicaSet with one member

Some operations involve deletion and recreation of MongoDB resource.

What can I do about it or how can I investigate it further? Logs does not say much, the first member simply cannot start after pod recreation.

**Comment by rodrigovalin** [comment]
Hello, thanks for reporting this issue.

There should be no operations that require the deletion of the MongoDB resources. Can you tell me what version of the operator are you using? I'll investigate this issue further.

**Comment by andrii-litvinov** [comment]
Hello, thanks for the response! I use Operator of version 1.4.2. I have tried to scale the cluster down and up from 2 nodes to 1 node and back, and it went fine. After I deleted the data from second member I cannot scale it up again. I can also give it a try with newest version of operator. The data size in the clusters varies between 5GB and 500GB.

**Comment by andrii-litvinov** [comment]
@rodrigovalin I have re-initiated my investigation after you response. I have deleted pod to see how cluster will behave and single-noded replica set was not able to successfully start after pod deletion with same RECOVERING stage. That gave me a hint that the issue is not with the Operator, but rather with with data in the cluster itself. So I googled a bit more and found suggestion to convert replica set to standalone and delete `local` db. That's basically what I did and then recreated MDB with ReplicaSet type again. It did solve the issue and I was able to scale my cluster to 3 members successfully.

So I guess when migrating from ShardedCluster to ReplicaSet (and renaming the replica set to remove `-0` suffix) I should have removed the `local` db completely instead of only removing records from `local.system.replset`, which somehow corrupted the data.

Thank you for your time! I will close the issue.
